components/style/properties/properties.mako.rs

```diff
@@ -1896,20 +1896,11 @@ impl ComputedValues {
         // TODO(gw): Add clip-path, isolation, mask-image, mask-border-source when supported.
         if effects.opacity < 1.0 ||
            !effects.filter.is_empty() ||
-           !effects.clip.is_auto() {
-           effects.mix_blend_mode != mix_blend_mode::T::normal ||
+           !effects.clip.is_auto() ||
+           effects.mix_blend_mode != mix_blend_mode::T::normal {
             return transform_style::T::flat;
```

Comment:
Yeah, the old wording was really confusing. I think the old logic could be rewritten as 
```rust
if effects.opacity < 1.0 ||
            !effects.filter.is_empty() ||
            !effects.clip.is_auto() {
            if effects.mix_blend_mode == mix_blend_mode::T::normal {
                return transform_style::T::flat;
            }
}
```
My assumption was that it's just a refactoring bug, where someone mixed the `||` with `{` at the end of the lines.